browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -3,6 +3,7 @@ const fs = require('fs')
 const path = require('path')
 const findStorage = require('browser/lib/findStorage')
 const mdurl = require('mdurl')
+const fse = require('fs-extra')
```

Comment:
Why do we need `fs-extra`? I assume `sander` can do the same job.

Reply:
I've used fs-extra because it has a good move-directory function. As far as i know sanders is not able to do so.. I think it contains some fallback implementation if the quick/efficient ways don't work..